File: app/src/main/kt/estyle/teabaike/activity/CopyrightActivity.java
package estyle.teabaike.activity;

import android.content.Context;
import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.view.MenuItem;

import estyle.teabaike.R;
import estyle.teabaike.databinding.ActivityCopyrightBinding;

public class CopyrightActivity extends BaseActivity {

    private ActivityCopyrightBinding binding;

    public static void startActivity(Context context) {
        Intent intent = new Intent(context, CopyrightActivity.class);
        context.startActivity(intent);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_copyright);

        initView();
    }

    private void initView() {
        setSupportActionBar(binding.toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                finish();
                break;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/kt/estyle/teabaike/activity/CopyrightActivity.kt
package estyle.teabaike.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.MenuItem
import estyle.teabaike.R
import kotlinx.android.synthetic.main.activity_copyright.*

class CopyrightActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_copyright)
        setSupportActionBar(toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> finish()
        }
        return super.onOptionsItemSelected(item)
    }

    companion object {

        fun startActivity(context: Context) {
            val intent = Intent(context, CopyrightActivity::class.java)
            context.startActivity(intent)
        }
    }
}
